src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/courtsentencing/CourtSentencingEventListener.kt

```diff
@@ -188,6 +193,15 @@ data class OffenderSentenceEvent(
   val auditModuleName: String?,
 )
 
+data class OffenderSentenceResynchronisationEvent(
+  val sentenceSeq: Int,
+  val dpsSentenceUuid: String,
```

Comment:
could you also send the consecutiveSentenceUuid so you don't need to do the look up? Simplify the code further

Reply:
will check